pkg/operator/ceph/cluster/crash/crash.go

```diff
@@ -122,57 +125,121 @@ func (r *ReconcileNode) createOrUpdateCephCrash(node corev1.Node, tolerations []
 	return controllerutil.CreateOrUpdate(context.TODO(), r.client, deploy, mutateFunc)
 }
 
-// createOrUpdateCephCron is a wrapper around controllerutil.CreateOrUpdate
-func (r *ReconcileNode) createOrUpdateCephCron(cephCluster cephv1.CephCluster, cephVersion *version.CephVersion) (controllerutil.OperationResult, error) {
-	cronJob := &v1beta1.CronJob{
+// createV1CronJob creates and return v1.CronJob for k8s version atleast than '1.21.0'
```

Comment:
would suggest renaming it to `getV1CronJob`  since this method is not creating the actual cronJob
```suggestion
// getV1CronJob returns v1.CronJob for k8s version higher than '1.21.0'
```

Reply:
changed to `// getV1CronJob returns v1.CronJob for k8s version atleast than '1.21.0'` as `1.21.0` is included